_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.md

```diff
@@ -3,59 +3,61 @@ layout: post
 
 ### Application Structure
 
-As covered previously, the aim is for automation (through XaaS or termination points) to be the overwhelmingly dominant method of communications between teams, especially for approvals. If even one team does not follow the standard of automating approvals, then deployment on demand, the ability to constantly deliver value to our clients, is dead in the water. Before cloud services, if a government employee wanted to have a virtual machine created in a data center they had to open tickets, create work request documentation, get approvals, send the work to another team after a series of handoffs, and so forth. With cloud technologies one is able to create the next generation virtualized technologies within minutes from anywhere in the world -- no coordination with other teams required. This is the same model of service that teams should offer to one another within the organization, with the added benefit of still being colleagues and being able to reach out for help, support, or have more say in future services that are to be implemented, as the internal service providers have a very limited set of customers, so should be able to provide very customized products to fit the needs of their internal users.
+As covered previously, the aim is for automation (through XaaS or termination points) to be the overwhelmingly dominant method of communications between teams, especially for approvals. If even one team does not follow the standard of automating approvals, then deployment on demand, the ability to constantly deliver value to our clients, is dead in the water. Before cloud services, if a government employee wanted to have a virtual machine created in a data center they had to open tickets, create work request documentation, get approvals, send the work to another team after a series of handoffs, and so forth. With cloud technologies one is able to create the next generation virtualized technologies within minutes from anywhere in the world -- no coordination with other teams required. This is the same model of service that teams should offer to one another within the organization, with the added benefit of still being colleagues and being able to reach out for help, support, or have more say in future services that are to be implemented.
 
-By creating a structure where the application team and the platform team are separate, this, via Conway's Law, will promote an architecture where the systems are loosely coupled and independent from one another. The team structure itself is intended to impact the architecture of the decision.
+Lastly, by creating a structure where the application team and the platform team are separate, this, via Conway's Law, will promote an architecture where the systems are loosely coupled and independent from one another. The team structure itself is intended to impact the architecture of the decision.
```

Comment:
What is the benefit of the systems being loosely coupled and independent?

Reply:
Pushed more context for this. It was in the conclusion, which is a bit late, but I also like ending with a quote. So wanted to keep it there. Let me know if you think this addresses the short coming